content/docs/cml/start-gitlab.md

```diff
@@ -0,0 +1,111 @@
+# Using CML on GitLab
+
+Here, we'll walk through a tutorial to start using CML on GitLab. For
+simplicity, we'll show the demo in GitLab Actions, but these instructions are
+valid for all supported Continuous Integration (CI) systems (with exceptions as
```

Comment:
I think we should remove the second sentence- I wanted a disclaimer like that on the project README, where I didn't want people to assume only GitHub was supported. Here, in the context of the docs and the title of this page, I think we can remove it and just get right to the insturctions.

Reply:
btw, probably `GitLab Actions` is not the right term?

Reply:
correct, it's GitLab CI and GitHub Actions